cloudigrade/account/tests/test_management_commands.py

```diff
@@ -1,11 +1,15 @@
 """Collection of tests for Account management commands."""
+import uuid
 from io import StringIO
+from unittest.mock import patch
+
 from django.core.management import call_command
-from django.utils.translation import gettext as _
 from django.test import TestCase
+from django.utils.translation import gettext as _
 
-
+from account.management.commands.add_account import aws
```

Comment:
are we importing from the right place here?

Reply:
Yes, with the way the mocking framework and Python imports work, you generally want to patch relative to where the code that needs the thing you're patching, not from where the thing originally lived. This is actually a common question (and can be a source of pain when getting started with patching), and they have a note about it here:

https://docs.python.org/3/library/unittest.mock.html#where-to-patch

> The basic principle is that you patch where an object is _looked up_, which is not necessarily the same place as where it is defined. A couple of examples will help to clarify this.